app/helpers/audits_helper.rb

```diff
@@ -42,6 +42,11 @@ def details(audit)
           (_("Provisioning Template content changed %s") % (link_to 'view diff', audit_path(audit))).html_safe if audit_template? audit
         elsif name == "owner_id" || name == "owner_type"
           _("Owner changed to %s") % (audit.revision.owner rescue _('N/A'))
+        elsif name == 'global_status'
+          base = audit.audited_changes.values[0]
+          from = HostStatus::Global.new(base[0]).to_label
+          to = HostStatus::Global.new(base[1]).to_label
+          _("Global status changed from %{from} to %{to}") % { :from => from, :to => to }
```

Comment:
Not sure what you mean exactly, global status is always driven by Foreman and can be only one of 0,1,2. Plugins may add sub-statuses that maps their statuses to one of global status. If e.g. some job invocation fails, it changes global status to 2 so you'll see that global status changed to error in audit log. But if you uninstall the plugin it does not affect global status.